docs/contributing.md

```diff
@@ -0,0 +1,40 @@
+# Contributing
+
+## Ways to contribute
+
+- Solve the issues
+- Suggest a feature
+- Submit new issues
+- Have fun and improve the DApp!
+
+## Follow the below steps to run the Application (WIP)
+
+1. Fork the repository
+2. Clone the repository locally  
+
+**Coming soon...**  
+
+
+### How the PR process works :building_construction
+
+:white_check_mark:
+Check for open issues
+
+:white_check_mark:
+Fork the repo
+
+:white_check_mark:
+Make some edits on your own copy that fixes the issues.
+
+:white_check_mark:
+Create a **new branch and then commit**.
+
+:white_check_mark:
+Create a pull request that will be merged to the main repo.
+(**Remember to mention which issue you're fixing**)
+
+:white_check_mark:Your code will be reviewed and approved.
```

Comment:
It should be, **The code will be reviewed and approved**

Reply:
Okay, Thanks!